### ISSUE TITLE
Trial a "sticky" position for nutrition facts imagery

### ISSUE THREAD

**Issue by CloCkWeRX** [openfoodfacts/openfoodfacts-server#2068]
With the current CSS; on some products (ie: https://world.openfoodfacts.org/cgi/product.pl?type=edit&code=9310081115144) the nutrition image gets fixed below the page scroll, or with relevant information obscured.

As a user, that means I have to scroll back and forth to ensure I'm keying in all of the relevant data.

Ideally, the image element would "stick" to above the page fold as a user scrolls through the nutrition edit section.

![image](https://user-images.githubusercontent.com/365751/60791754-73706e00-a1a3-11e9-9253-5cbf9bb76f69.png)

I managed this one with:
 * Nutrition table becomes
```
<table id="nutrition_data_table" class="data_table" style="display: block; max-height: 600px; overflow-y: auto">
```
 * Image becomes 
```
<div id="nutrition_image_copy" style="position: absolute; top: 50px; left: 506px;"><img src="/images/products/931/008/111/5144/nutrition_en.9.400.jpg"></div>
```

However, it currently feels like it "steals" the mouse/scrolling too easily.

Alternative implementations might involve position: sticky (https://developer.mozilla.org/en-US/docs/Web/CSS/position) or only adding this behaviour when the table elements have :focus somehow.

Wanted to see what people thought before throwing in a PR; or if there are better design / layout approaches to solve the usabilty annoyance.

**Comment by CharlesNepote** [comment]
I tried your solution (implemented it in a Power User Script test version). Your issue is perfectly relevant and your idea is very good but I dislike the lift bar at the right of the image (as we can see in your screenshot). There shouldn't be any lift at all. I think that's what you're saying here: "However, it currently feels like it "steals" the mouse/scrolling too easily."

That said, you're probably not so far from a good solution :) My CSS skills are not very good sorry I can't help you...

**Comment by CloCkWeRX** [comment]
Yeah, it didn't feel right. Leaving this open to explore alternative UX approaches though instead.

**Comment by advitiya7** [comment]
is this issue open ?

**Comment by alexgarel** [comment]
@advitiya7 it's opened and @CharlesNepote is eager to have it :-)

You can see code doing that in https://github.com/openfoodfacts/power-user-script/
